examples/specs/bar_custom_sort_partial.vl.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A simple bar chart with embedded data.",
+  "data": {
+    "values": [
+      {"a": "A","b": 28}, {"a": "B","b": 55}, {"a": "C","b": 43},
+      {"a": "D","b": 91}, {"a": "E","b": 81}, {"a": "F","b": 53}
```

Comment:
Actually, change this to Z, Y, X? 

I believe the result would be B, A, C, Z, Y, X.       
(The reason we should change is that using  B, A, C, D, E, F, it won't be clear whether it's D E F because of the original order or because D is naturally before E and F.)

Reply:
(Please double check to make sure I'm not crazy :p)